opendebates/templates/opendebates/changelog.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+<div class="row breadcrumbs">
+  <a href="/">{% blocktrans %}Home{% endblocktrans %}</a>
+  <span class="hidden-xs">&#8594;</span>
+  <a class="hidden-xs" href="">{% blocktrans %}Changelog{% endblocktrans %}</a>
+</div>
+<div>
+  <div class="table-responsive">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Question</th>
+          <th>Date &amp; Time</th>
+          <th>Change Made</th>
+          <th>Information</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for idea in moderated %}
```

Comment:
Do we need to be paginating this?  (Looks like we've got at least ~2000 entries here on presidentialopenquestions.com I think.)